cockroachcloud/cloud-api.md

```diff
@@ -390,6 +390,46 @@ Where:
 
 If the `DELETE` request was successful the client will not receive a response payload.
 
+<a id="cloud-audit-logs"></a>
+
+## Export {{ site.data.products.db }} audit logs
+
+{% include feature-phases/preview-opt-in.md %}
+
+To export audit logs for activities and events related to your {{ site.data.products.db }} organization, send a `GET` request to the `/v1/auditlogevents` endpoint. The service account associated with the secret key must have `ADMIN` [permission](console-access-management.html#service-accounts).
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+curl --request GET \
+  --url 'https://cockroachlabs.cloud/api/v1/auditlogevents?StartingFrom={timestamp},SortOrder={sort_order},limit={limit}' \
+  --header 'Authorization: Bearer {secret_key}'
+~~~
+
+Where:
+
+  - `{timestamp}` indicates the first log entry to fetch. If unspecified, `{timestamp}` is set to the beginning of time if `{sort_order}` is `ascending`, or the current time if `{sort_order}` is `descending`.
```

Comment:
Again, should contextualize "beginning of time" in terms of the lifetime of an organization and our retention policy (if any), based on @rgcase 's response to similar comments in other file.

Reply:
There's no log expiry at the moment, but if we want to choose a retention policy where they expire we can. At the moment the default will be the beginning of time, or the create time of the organization, whichever is later :).

Reply:
Got it, thanks Ryan. Let's discuss retention later then.

@mdlinville - In that case, I think we should document that "beginning of time" refers to the creation of a cloud organization.